Social-Music/Singletons/CustomRefresh.m

```diff
@@ -0,0 +1,32 @@
+//
+//  CustomRefresh.m
+//  Social-Music
+//
+//  Created by Archita Singh on 8/2/22.
+//
+
+#import "CustomRefresh.h"
+
+@implementation CustomRefresh
++ (id)shared {
+    static CustomRefresh *shared = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        shared = [[self alloc] init];
```

Comment:
Again, you don't need this as a singleton, you can just init the object where you need it.

[[[CustomRefresh alloc] init] customRefresh:...];